charts/plex-media-server/Chart.yaml

```
@@ -22,7 +22,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.4.0
+version: 0.4.1
```

nit: new feature should be a minor bump

```suggestion
version: 0.5.0
```